scripts/Connector/Connector_TE-Connectivity/conn_te_928492.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+
+# KicadModTree is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# KicadModTree is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kicad-footprint-generator. If not, see < http://www.gnu.org/licenses/ >.
+#
+# (C) 2016 by Thomas Pointhuber, <thomas.pointhuber@gmx.at>
+
+"""
+
+Datasheet not found available online
+See https://github.com/KiCad/kicad-footprints/files/3331008/ENG_CD_928492_B3.pdf
+
+Generates only the gray color part numbers
+Footprints are the same, but different colors have unique keying
+Thus 3D models would be unique
+
+"""
+
+import sys
+import os
+#sys.path.append(os.path.join(sys.path[0],"..","..","kicad_mod")) # load kicad_mod path
+
+# export PYTHONPATH="${PYTHONPATH}<path to kicad-footprint-generator directory>"
+sys.path.append(os.path.join(sys.path[0], "..", "..", ".."))  # load parent path of KicadModTree
+from math import sqrt
+import argparse
+import yaml
+from helpers import *
+from KicadModTree import *
+
+sys.path.append(os.path.join(sys.path[0], "..", "..", "tools"))  # load parent path of tools
+from footprint_text_fields import addTextFields
+
+manufacturer = "TE-Connectivity"
+partnumber = "928492"
+datasheet = "https://github.com/KiCad/kicad-footprints/files/3331008/ENG_CD_928492_B3.pdf"
+
+pincounts = [2,4,6,7,8,9,11,12]
+
+def generate_one_footprint(pincount, configuration):
+    pitch = 5
+    drill = [1.2, 1.1] # pin row closest to open end is 1.1mm and back row is 1.2mm
```

Comment:
Just to be (absolutely) safe, you corrected the DF11 footprints wrt to tolerances.
For both it says +-0.05mm, resulting in 1.225 and 1.125.
Is this too exact?

Reply:
@herostrat 
We should center our pads in the tolerance range. For an equal plus and minus tolerance, the nominal value is in the middle and thus the size to use. We only deviate from the nominal value if the tolerance range is unequal. I think this is correct so I feel I'm missing your point. Can you help me understand?

Reply:
ah ok, I read that wrong.
Sorry for the false positive.